src/slider.tsx

```diff
@@ -0,0 +1,262 @@
+import * as React from 'react';
+import {useScreenSize, useTheme} from './hooks';
+import * as styles from './slider.css';
+import classnames from 'classnames';
+import IntegerField from './integer-field';
+import Tooltip from './tooltip';
+
+interface SliderProps {
+    disabled?: boolean;
+    steps?: number | Array<number>;
+    max?: number;
+    min?: number;
+    field?: boolean;
+    value?: number;
+    onChange?: (value: number) => void;
+    getStepArrayIndex?: (value: number) => void;
+    invalidText?: string;
+    'arial-label'?: string;
+}
+
+const Slider: React.FC<SliderProps> = ({
+    disabled,
+    steps = 1,
+    max = 100,
+    min = 0,
+    field,
+    value,
+    onChange,
+    getStepArrayIndex,
+    invalidText = ' ',
```

Comment:
follow the same approach as TextField input

Add helperText (string) & error (boolean)
https://tinyurl.com/28gka68m